repos/system_upgrade/el7toel8/actors/satellite_upgrade_facts/tests/unit_test_satellite_upgrade_facts.py

```diff
@@ -84,6 +86,20 @@ def test_enables_pki_modules(current_actor_context):
     assert Module(name='pki-deps', stream='10.6') in message.modules_to_enable
 
 
+def test_enables_satellite_module(current_actor_context):
+    current_actor_context.feed(InstalledRPM(items=[FOREMAN_RPM, SATELLITE_RPM]))
+    current_actor_context.run(config_model=mock_configs.CONFIG)
+    message = current_actor_context.consume(RpmTransactionTasks)[0]
+    assert Module(name='satellite', stream='el8') in message.modules_to_enable
```

Comment:
I mean since they are mutually exclusive as per code at least, maybe you'd want to check the negative as well
```suggestion
    assert Module(name='satellite', stream='el8') in message.modules_to_enable
    assert Module(name='satellite-capsule', stream='el8') not in message.modules_to_enable
```

same just swapped for `test_enables_satellite_capsule_module`

Reply:
good point! updated